### DISCUSSION TITLE
[FEATURE] Logo Design

### DISCUSSION THREAD

**Discussion by YuriDevAT** [YurisCodingClub/pro-check#11] [Ideas]
### Description

How should the Logo of the app look like?
Let's collect some ideas (maybe from [Behance](https://www.behance.net/) or [dribbble](https://dribbble.com/)) and choose with which kind of design we want to go further.

- [ ] Design for the Logo
- [ ] Depending on the Logo, design for an Icon

_Old issue_

> 
> Create a Logo DevCV which will be showcased later on in the Header and/or Footer as well as Icon.
> 
> ### Acceptance Criteria
>  
> - [ ] The Logo's color is accessible. For more information check out [WebAIM's article about color contrast(https://webaim.org/articles/contrast/).
> - [ ] The Logo is saved as SVG called logo.svg
> - [ ] The Icon is saved as favicon.ico
> - [ ] A folder is created in the public directory and called "images"
> - [ ] The Logo is saved in the new folder called "images"
> - [ ] The old favicon from react is replaced by the new created favicon
> 
> ### Screenshots
> 
> _No response_
> 
> ### Additional information
> 
> If you do not know how to use git, no worries. I will help you with it to make sure you get the Hacktoberfest-accepted badge.

**Comment by areesha-sayed** [comment]
Can you assign this one to me? I would love to work on it.

**Comment by YuriDevAT** [comment]
I do like these kind of style of logo (gradient) and the combination with the name:

[See the whole logo design project](https://www.behance.net/gallery/153535893/Logo-Logo-design-Branding?tracking_source=search_projects%7Clogo+design)
<img width="559" alt="Screenshot 2022-10-15 at 10 03 48" src="https://user-images.githubusercontent.com/54622834/195976347-a76a6671-9773-42a5-921a-66421f463e66.png">

[See the whole logo design project](https://www.behance.net/gallery/154653825/Tech-Logo-Brand-Identity-Logo-Design?tracking_source=search_projects%7Clogo+design)
<img width="1077" alt="Screenshot 2022-10-15 at 10 04 14" src="https://user-images.githubusercontent.com/54622834/195976365-7a80231e-a865-43e4-a94f-26f3a6063017.png">

[See the whole logo design project](https://dribbble.com/shots/19299319-logo-logo-design-branding-logotype-brand-identity) 
<img width="982" alt="Screenshot 2022-10-15 at 10 03 00" src="https://user-images.githubusercontent.com/54622834/195976292-b63b5275-f427-4404-ab57-ee62eea52a00.png">
